src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -353,6 +354,23 @@ bool emberAfTestClusterClusterTestListStructArgumentRequestCallback(
 
     return SendBooleanResponse(commandObj, commandPath, shouldReturnTrue);
 }
+bool emberAfTestClusterClusterTestEmitTestEventRequestCallback(
+    chip::app::CommandHandler * commandObj, const chip::app::ConcreteCommandPath & commandPath,
+    const chip::app::Clusters::TestCluster::Commands::TestEmitTestEventRequest::DecodableType & commandData)
+{
+    Commands::TestEmitTestEventResponse::Type responseData;
+    DataModel::List<const Structs::SimpleStruct::Type> arg5;
+    DataModel::List<const SimpleEnum> arg6;
```

Comment:
Why not pass these in the command payload?

Reply:
I'd recommend using `NullablesAndOptionalsStruct` since it covers lists as well as nullables/optionals, and embedding that in both the command and the event.

Reply:
That was the original pattern I was shooting for, but the interface to go from DecodableType to Type proved to be marginally more complicated, so I mocked up something temporary. Will fix.